x-pack/plugins/observability/public/components/custom_threshold/components/custom_equation/metric_row_controls.tsx

```diff
@@ -15,19 +15,15 @@ interface MetricRowControlProps {
 
 export function MetricRowControls({ onDelete, disableDelete }: MetricRowControlProps) {
   return (
-    <>
-      <EuiFlexItem grow={0}>
-        <EuiButtonIcon
-          data-test-subj="o11yMetricRowControlsButton"
-          aria-label={DELETE_LABEL}
-          iconType="trash"
-          color="danger"
-          style={{ marginBottom: '0.6em' }}
-          onClick={onDelete}
-          disabled={disableDelete}
-          title={DELETE_LABEL}
-        />
-      </EuiFlexItem>
-    </>
+    <EuiButtonIcon
+      data-test-subj="o11yMetricRowControlsButton"
+      aria-label={DELETE_LABEL}
+      iconType="cross"
+      color="text"
+      onClick={onDelete}
+      disabled={disableDelete}
+      title={DELETE_LABEL}
+      size="xs"
```

Comment:
I suggest changing the icon size to 16 instead of 24 to avoid the jump as shown below:

Size 24

https://github.com/elastic/kibana/assets/12370520/fc307883-75a2-4a24-9c38-74f13524a74b

Size 16

https://github.com/elastic/kibana/assets/12370520/76cb3ec6-f9a1-4571-99f7-df9a7f160e5e

When I used size="xxs" it worked, but it seems EuiButtonIcon does not have xxs its size 🙃 

Alternatively, we can set the height of the Aggregation name to 24px.